workflows/crisprseq_screening.nf

```diff
@@ -357,10 +359,23 @@ workflow CRISPRSEQ_SCREENING {
 
     }
 
+    //
+    // Venn diagrams
+    //
 
+    // BAGEL2 and MAGeCK MLE
     if(params.mle && params.bagel2) {
         ch_venndiagram = BAGEL2_PR.out.pr.join(MAGECK_MLE.out.gene_summary)
+        ch_venndiagram.dump(tag: "Venn diagram")
         VENNDIAGRAM(ch_venndiagram)
+        ch_versions = ch_versions.mix(VENNDIAGRAM.out.versions)
+    }
+
+    if(params.mle && params.drugz) {
+        ch_venndiagram = DRUGZ.out.per_gene_results.join(MAGECK_MLE.out.gene_summary)
```

Comment:
can bagel2 and drugz both run in the same analysis? We should give the channel a different name to avoid overriding problems.